css-responsive-layout/styles.css

```diff
@@ -0,0 +1,47 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+}
+
+.fighter {
+  width: 100%;
+}
+
+.container {
+  max-width: 992px;
+  margin: 0 auto;
+  padding: 0 1rem;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.col,
+.col-sm-half,
+.col-lg-third,
+.col-lg-fourth {
+  padding: 0 0.5rem;
+  flex-basis: 100%;
+}
+
+@media only screen and (width >= 576px) {
+  .col-sm-half {
+    flex-basis: calc(100% / 2);
+  }
+}
+
```

Comment:
For something like this, since we know that 100% / 2 = 50%, we can just use `flex-basis: 50%`